buildSrc/src/main/resources/uber-plugin-descriptor.properties

```diff
@@ -0,0 +1,25 @@
+# Elasticsearch uber plugin descriptor file
+# This file must exist as 'uber-plugin-descriptor.properties' in a folder named `elasticsearch`.
+#
+### example uber plugin for "uber-foo"
+#
+# uber-foo.zip <-- zip file for the uber plugin, with this structure:
+#|____elasticsearch/
+#| |____   <sub_plugin_1> <-- The plugin files for sub_plugin_1 (the content of the elastisearch directory)
+#| |____   <sub_plugin_2>  <-- The plugin files for sub_plugin_2
+#| |____   uber-plugin-descriptor.properties <-- example contents below:
+#
+# description=My uber plugin
+# name=uber-foo
+# plugins=sub_plugin_1,sub_plugin_2
+#
+### mandatory elements for all uber-plugins:
+#
+# 'description': simple summary of the uber plugin
+description=${description}
+#
+# 'name': the uber plugin name
+name=${name}
+#
+# 'plugins': a coma separated list of the sub-plugin names bundled in the uber plugin
```

Comment:
typo: coma -> comma

Also, I would not call these "sub" plugins (that has a connotation of subclassing), but instead "bundled" plugins?

Reply:
But is this plugins list actually needed? The loading doesn't use it (and shouldn't, no leniency for extra junk at the root). I don't see it used anywhere but in tests.

Reply:
Agreed, I removed it.